src/autolabel/models/hf_pipeline.py

```diff
@@ -8,6 +9,11 @@
 from autolabel.schema import RefuelLLMResult
 
 
+logger = logging.getLogger(__name__)
+
+DEFAULT_NUM_BEAMS = 4
```

Comment:
Is this constant being used anywhere? If not, let's remove it.

Reply:
There's a num_beams parameter in huggingface that is generally recommended to be used with sequence biasing but it didn't make a difference in testing for me so I'll remove it